modules/volume/src/algorithm/volumemap.cpp

```diff
@@ -43,6 +43,12 @@ void remap(std::shared_ptr<Volume>& volume, std::vector<int> src, std::vector<in
                         src.size(), dst.size());
     }
 
+    auto uniqueIterator = std::unique(src.begin(), src.end());
```

Comment:
std::unique requires that the input range is sorted, is that the case?

Reply:
Did not know that. I just pushed a change where I create a sorted copy of src, and use std::unique for the copy. Of course, it requires an extra sort, but maybe that's not too bad, considering it only has to run once when changing CSV file?

Reply:
Could use `std::is_sorted()` (see [docs](https://en.cppreference.com/w/cpp/algorithm/is_sorted)) or even `std::is_sorted_until()`.